### PR TITLE
[HOTFIX] students not in team

### PR DESCRIPTION
**Pull request by akbarsahata**
## Changes Notes

- [FIX] issues of returning same user and returning user data with their own id

### CODE REVIEW

**Review by azhariemuhammad — APPROVED** [review]
LGTM 👍🏿

File: src/api/services/user.ts [hunk 1]
```diff
@@ -159,8 +165,14 @@ export default class UserService {
       );
     }
 
-    if (name) {
-      userQuery = userQuery.where('user.name ILIKE :name', {name: `%${name}%`});
+    if (params?.name) {
+      userQuery = userQuery
+        .where('user.name ILIKE :name', {name: `%${params.name}%`})
```
Comment (yofriadi) [inline src/api/services/user.ts:170]: eh `ILIKE` sql emang?

Comment (akbarsahata) [reply src/api/services/user.ts:170]: Proprietary nya postgre

Comment (yofriadi) [reply src/api/services/user.ts:170]: oalah mantap senior gua emang

**Pull request merged**